practice-two/src/stores/reducers.js

```diff
@@ -0,0 +1,30 @@
+import { ACTION_TYPES } from '@constants';
+
+export const initialCustomer = {
+  customers: [],
+};
+
+export const customerReducer = (state, action) => {
+  switch (action.type) {
```

Comment:
destruct object states

Reply:
and action

Reply:
![image](https://github.com/datdragon777/React-Training/assets/74806826/ae1c8f49-f9ae-48b2-a6d6-552f44b77ffa)